src/execution_plan/ops/op_delete.c

```diff
@@ -177,6 +178,22 @@ static Record DeleteConsume(OpBase *opBase) {
 			array_append(op->deleted_edges, *e);
 			// If evaluating the expression allocated any memory, free it.
 			SIValue_Free(value);
+		} else if(type & T_PATH) {
+			Path *p = (Path *)value.ptrval;
+			size_t nodeCount = Path_NodeCount(p);
+			size_t edgeCount = Path_Len(p);
```

Comment:
```c
size_t edgeCount = Path_EdgeCount(p);

for(size_t i = 0; i < nodeCount; i++) {
    Node *n = Path_GetNode(p, i);
    array_append(op->deleted_nodes, *n);
}
for(size_t i = 0; i < edgeCount; i++) {
    Edge *e = Path_GetEdge(p, i);
    array_append(op->deleted_edges, *e);
}
```

Reply:
Thanks. I saw that @AviAvni fixed it in c1ad5f6